addons/account/static/src/components/actionable_errors/actionable_errors.js

```diff
@@ -0,0 +1,53 @@
+/** @odoo-module **/
+
+import { registry } from "@web/core/registry";
+import { Component, xml } from "@odoo/owl";
+import { standardFieldProps } from "@web/views/fields/standard_field_props";
+import { useRecordObserver } from "@web/model/relational_model/utils";
+
+export class ActionableErrors extends Component {
+    static props = { ...standardFieldProps };
+    static template = xml`
+    <t t-name="account.ActionableErrors">
```

Comment:
🤔 sorry, I think I'm missing something. I'm not sure why the template has to be inlined

Reply:
This PR is for stable, 17.0

Once the PR is merged, the code gets updated, but the views and the templates in  `account` and `l10n_it_edi` modules will still need upgrading.

Once `l10n_it_edi` is upgraded, the modified Send&Print view will need to show the widget. The JS file with the widget is already upgraded, but the eventual XML file with the template would not be loaded and found until the `account` module is upgraded, so until you do it would not find it and traceback.

In master we can of course have the separate file.

Reply:
Well, I retested, what you say is True, so I take everything back. Done.